exp-player/addon/components/exp-exit-survey.js

```diff
@@ -148,7 +147,7 @@ export default ExpFrameBaseComponent.extend({
                 Ember.merge(formData, this.getValue());
                 root.set('formData', formData);
                 console.log('Post-study survey complete.');
-                root.actions.next();
+                root.actions.next.apply(root);
```

Comment:
Instead of `root.actions.next.apply`, use `root.send('actionName', arguments)`
http://emberjs.com/api/classes/Ember.Component.html#method_send

(Or `sendAction` for an action that was passed in and does not live in this controller)